client/my-sites/checkout/composite-checkout/components/wp-checkout.tsx

```diff
@@ -537,6 +537,11 @@ const CheckoutSummaryBody = styled.div`
 		max-width: 328px;
 		position: fixed;
 		width: 100%;
+		overflow-y: auto;
+		height: 100vh;
+	}
+	.card {
+		border: 1px solid ${ ( props ) => props.theme.colors.borderColorLight };
```

Comment:
This looks good at wider widths, but at thinner widths, it looks very thick:

<img width="579" alt="Screenshot 2022-12-07 at 5 57 07 PM" src="https://user-images.githubusercontent.com/2036909/206315369-88ab63bd-3043-4764-b946-ff765c9e8a73.png">

Compare this to how it looks in production:

<img width="574" alt="Screenshot 2022-12-07 at 6 00 25 PM" src="https://user-images.githubusercontent.com/2036909/206315479-51135bf2-c480-4e18-b8c7-2e3c66ea8df3.png">

Honestly, the placement is awkward even before this PR and maybe we should remove the spacing around it at small widths too, but that's beyond the scope of this change.

Reply:
That is correct, looks thick. I have done some changes as it looks like it already had a border top and bottom. Now it looks close to production:

![g6Vz6R.png](https://user-images.githubusercontent.com/552016/206558446-4211fddd-e39d-48b4-aa84-359083a88ded.png)

Reply:
That screenshot is desktop width. Did you check mobile width? That's what I'm talking about.

Reply:
oh ok, got it, I think it looks quite close now on the mobile layout:

![XR0Uba.png](https://user-images.githubusercontent.com/552016/206583699-0a82b1b5-fc33-4dd1-ac89-7eb35b1f2550.png)